metrics/influxdb.conf

```diff
@@ -0,0 +1,14 @@
+[meta]
```

Comment:
It may be worth specifying the file extension as `*.toml` instead of `*..conf` or it was done intentional?

Reply:
InfluxDB simply wants it to be called `influxdb.conf`: https://docs.influxdata.com/influxdb/v1.8/administration/config/#configuration-overview